DMDisassembler/Program.cs

```diff
@@ -31,31 +31,50 @@ internal class Program {
         LoadAllTypes();
 
         bool acceptingCommands = true;
+
+        // Commands passed as CLI arguments
+        if (args.Length > 1) {
+            ParseCommand(args[1..], out acceptingCommands);
```

Comment:
Allowing a single command to be executed from the command line doesn't make much sense for most of these. I'd rather this just check for a `crash-on-error` argument instead of making it a command.